static/styles/zulip.css

```diff
@@ -1607,7 +1607,7 @@ div.focused_table {
 }
 
 .messagebox-content {
-    padding: 4px 115px 1px 10px;
+    padding: 4px 80px 1px 10px;
 }
```

Comment:
This seems like it affects all messages, not just drafts, and thus is likely to break message formatting.